content/models.py

```diff
@@ -27,3 +27,17 @@ class OurStoryTitle(models.Model):
 
     def __str__(self):
         return self.title
+
+class ChildrensProgram(models.Model):
+	pagetitle = models.CharField(blank=True, null=True, max_length=100)
```

Comment:
pagetitle: what we do childrens program, subtitle : art, music etc 
![screenshot from 2016-03-05 17 38 20](https://cloud.githubusercontent.com/assets/10239547/13546686/1f20b3d0-e2f9-11e5-8911-077ef794b317.png)